scripts/reduce_sql.py

```diff
@@ -19,6 +19,28 @@
 '''
 
 
+class MultiStatementManager:
```

Comment:
Unrelated to this PR specifically - there are copies of these files in the fuzzer ci repo (https://github.com/duckdblabs/duckdb-fuzzer-ci) - perhaps we can remove those and make it download the files from the main DuckDB repo instead

Reply:
My preference (given sqlsmith will go out of tree), would be having all fuzzer related logic moved out of tree, either  to say duckdb/duckdb_sqlsmith or equivalent name, or even to duckdblabs/duckdb-fuzzer-ci.

Idea would be that there is some overlap with duckdb syntax, but this is mostly good across duckdb-versions.